packages/realm/src/app-services/User.ts

```diff
@@ -177,7 +198,7 @@ export class User<
    * Use this to call functions defined by the Atlas App Services application, as this user.
    */
   get functions(): FunctionsFactoryType {
-    throw new Error("Not yet implemented");
+    return createFactory(this as User, undefined);
```

Comment:
Why do you need the `this as User` cast? I thought that would be implied on any method of User. Or is this to strip off the template params?